addons/stock/models/stock_picking.py

```diff
@@ -1084,6 +1086,8 @@ def _log_activity_get_documents(self, orig_obj_changes, stream_field, stream, so
         for (parent, responsible), moves in grouped_moves:
             if not parent:
                 continue
+            if whitelist and parent not in [rec[1] for rec in whitelist]:
+                continue
```

Comment:
i don't get the whitelist arg, and it doesn't seem used?

Reply:
yep it was on the previous version. I delete it